File: API/Models/PerfilUsuarioListaModel.cs
namespace API_Visitatus.Models
{
    public class PerfilUsuarioListaModel
    {
        public int peUCodi { get; set; }
        public bool peUStat { get; set; }
        public int perCodi { get; set; }
        public long usuCodi { get; set; }
        public long lojCodi { get; set; }
        public string? perNome { get; set; }
        public bool perStat { get; set; }
        public string? lojNome { get; set; }
        public string? lojNumL { get; set; }
        public bool lojStat { get; set; }
        public int potCodi { get; set; }
    }
}

File: Sistema/src/app/models/PerfilUsuarioLista.Model.ts
export class PerfilUsuarioListaModel {
  peUCodi: number;
  peUStat: boolean;
  perCodi: number;
  usuCodi: number;
  lojCodi: number;
  perNome: string;
  perStat: boolean;
  lojNome: string;
  lojNumL: string;
  lojStat: boolean;
  potCodi: number;

  constructor() {
    this.peUCodi = 0;
    this.peUStat = false;
    this.perCodi = 0;
    this.usuCodi = 0;
    this.lojCodi = 0;
    this.perNome = '';
    this.perStat = false;
    this.lojNome = '';
    this.lojNumL = '';
    this.lojStat = false;
    this.potCodi = 0;
  }
}
